src/Esprima/Ast/AccessorProperty.cs

```diff
@@ -1,27 +1,34 @@
+using System.Runtime.CompilerServices;
 using Esprima.Utils;
 
 namespace Esprima.Ast;
 
 public sealed class AccessorProperty : Node
 {
-    /// <summary>
-    /// <see cref="Expression" /> | <see cref="PrivateIdentifier" />
-    /// </summary>
-    public readonly Expression Key;
-    public readonly Expression? Value;
-    public readonly bool Computed;
-    public readonly bool Static;
-    public readonly NodeList<Decorator> Decorators;
+    private readonly Expression _key;
+    private readonly Expression? _value;
+    private readonly bool _computed;
+    private readonly bool _static;
+    private readonly NodeList<Decorator> _decorators;
 
     public AccessorProperty(Expression key, Expression? value, bool computed, bool isStatic, in NodeList<Decorator> decorators) : base(Nodes.AccessorProperty)
     {
-        Key = key;
-        Value = value;
-        Computed = computed;
-        Static = isStatic;
-        Decorators = decorators;
+        _key = key;
+        _value = value;
+        _computed = computed;
+        _static = isStatic;
+        _decorators = decorators;
     }
 
+    /// <remarks>
+    /// <see cref="Expression" /> | <see cref="PrivateIdentifier" />
+    /// </remarks>
+    public Expression Key { [MethodImpl(MethodImplOptions.AggressiveInlining)] get => _key; }
```

Comment:
For a shorter syntax without need for backing field you could also try:

```csharp
public Expression Key { [MethodImpl(MethodImplOptions.AggressiveInlining)] get; }
```
It didn't change benchmark results and resharper/rider allows you to do a mass-change with alt-enter and selecting suitable scope, like folder.

Reply:
You're right, I've changed this.